dojo/settings/settings.dist.py

```diff
@@ -1468,7 +1468,7 @@ def saml2_attrib_map_format(dict):
     'kube-bench Scan': DEDUPE_ALGO_HASH_CODE,
     'Threagile risks report': DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL_OR_HASH_CODE,
     'Humble Json Importer': DEDUPE_ALGO_HASH_CODE,
-    'MSDefender API': DEDUPE_ALGO_HASH_CODE,
+    'MSDefender Parser': DEDUPE_ALGO_HASH_CODE,
```

Comment:
There is something wrong with your PR. Please fix your commit history.

Reply:
+1 ^

Reply:
fixed, thank you

Reply:
No, it is not fixed.

Reply:
sorry, @manuel-sommer , what should i do to fix commit history? I thought I just should rebase to DefectDojo:dev ...
or may be should i squash my several commits into single?

Reply:
This link will probably help: 
https://github.com/openedx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

Reply:
tried to rebase as described in this manual, I hope now it's ok